x-pack/legacy/plugins/epm/server/datasources/create.ts

```diff
@@ -97,10 +109,30 @@ async function getDatasource(options: {
   return datasource?.attributes;
 }
 
-function createFakeDatasource(pkg: RegistryPackage, assets: Asset[] = []): Datasource {
+function createFakeDatasource(
+  pkg: RegistryPackage,
+  datasourceName: string,
+  datasetsToInstall: Dataset[],
+  assets: Asset[] = []
+): Datasource {
+  const streams = datasetsToInstall.map(stream => ({
```

Comment:
Not an issue for this PR at all, but I think `stream` should be `dataset`. The. type is `Dataset`, even if we're returning a value to be used in/as a stream.